src/native/tools/ssl_napi.cpp

```diff
@@ -127,7 +133,8 @@ typedef Hasher<
 
 typedef Hasher<
     SHA1_MB_ALG,
-    SHA1_DIGEST_NWORDS * 4,
+    SHA1_DIGEST_NWORDS,
+    true, // Digest words for SHA1 are big-endian
```

Comment:
Can you open an issue to ask on this for isa-l_crypto?
It is unclear why this is the chosen behavior right?

Reply:
opened https://github.com/intel/isa-l_crypto/issues/57